index.html

```diff
@@ -16,7 +16,7 @@
     <title>portfolio</title>
   </head>
 
-  <body>
+  <body class="body">
```

Comment:
- [ ] Dear @Tracy-miranja, Your design needs some arrangements when it comes to being responsive on a different screen.  As you can see It has a horizontal scroll which is not a good thing regarding user experience.

![Screenshot 2022-12-13 165952](https://user-images.githubusercontent.com/69077061/207351408-55ec5c0d-2f61-4e17-9af4-e84620aac479.png)